src/components/role-category/role-category.json

```diff
@@ -0,0 +1,20 @@
+{
+  "collectionName": "components_role_category_role_categories",
+  "info": {
+    "displayName": "roleCategory",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "category": {
+      "type": "enumeration",
+      "enum": [
+        "Product / UX",
+        "Development",
+        "QA",
+        "Operations",
+        "Other"
+      ]
+    }
+  }
+}
```

Comment:
`roleType` is the previous column used by the opportunity collection that will be replaced by the roleCategory. My plan is:
1. Merge and deploy roleCategory
2. Set roleCategory to be the same value as roleType
3. Delete roleType
Ideally we can write a migration to automatically set roleCategory to the same value as roleType. However, given that we only have a few opportunities in the database, it's faster to do the migration manually.

Reply:
Hmm maybe it is something like a subcategory ? 

Let me know if it is correct. 
We can create a model roleCategory that has the attribute category.
But to make it practical we can use it as a component ?

It is possible to cre

Reply:
It's not a subcategory, but a new type. Under this new type, there is an attribute called category.
We make this type into a component, so it can be reused between collections.